cscs-checks/tools/profiling_and_debugging/scorep_mpi_omp.py

```diff
@@ -12,10 +12,12 @@ def __init__(self, lang):
         self.name = 'scorep_mpi_omp_%s' % lang.replace('+', 'p')
         self.descr = 'SCORE-P %s check' % lang
         self.valid_systems = ['daint:gpu', 'daint:mc', 'dom:gpu', 'dom:mc']
-        self.valid_prog_environs = ['PrgEnv-gnu', 'PrgEnv-intel', 'PrgEnv-pgi',
-                                    'PrgEnv-cray']
+        self.valid_prog_environs = ['PrgEnv-gnu', 'PrgEnv-intel', 'PrgEnv-pgi']
+        # scorep fails with latest clang based cce compiler:
+        # src/measurement/thread/fork_join/scorep_thread_fork_join_omp.c:402:
+        # Fatal: Bug 'TPD == 0': Invalid OpenMP thread specific data object.
+        # -> removing cce from supported compiler for now.
```

Comment:
I would put the comment above the `valid_prog_environs`. @jgphpc Downgrading the compiler didn't solve the issue?

Reply:
@jgphpc As soon as you address this comment, this PR will be merged.